target/linux/ipq40xx/image/generic.mk

```diff
@@ -336,7 +338,7 @@ define Device/cilab_meshpoint-one
 	$(call Device/8dev_jalapeno-common)
 	DEVICE_VENDOR := Crisis Innovation Lab
 	DEVICE_MODEL := MeshPoint.One
-	DEVICE_PACKAGES := kmod-i2c-gpio kmod-iio-bmp280-i2c kmod-hwmon-ina2xx kmod-rtc-pcf2127
+	DEVICE_PACKAGES += kmod-i2c-gpio kmod-iio-bmp280-i2c kmod-hwmon-ina2xx kmod-rtc-pcf2127
```

Comment:
I think this should be done in separate commit as it might seems that you are doing something else than what is said in the commit subject and commit description

Reply:
You are right, I failed to explicitly mention MeshPoint.One thinking it as subclass of Jalapeno. Updated now.